### ISSUE TITLE
Link Insert Incorrect Placement

### ISSUE THREAD

**Issue by macrodesignuk** [Frontwise/grid-editor#34]
I was testing this and found when selecting a word in the content area to convert into a hyper link, the word is copied and inserted again at the beginning of the content area and turned into a link instead of converting the pre-selected word as a link. This also happens when trying to insert an image in a specific spot, it goes straight to the start.

e.g if you try and make 'fox' a hyper link
> The quick brown fox jumps over the lazy dog

it becomes
>**fox** The quick brown fox jumps over the lazy dog

**Comment by Epskampie** [comment]
Thanks for reporting this.

However, this seems to be more of an issue with the rich text editor. What version where you testing?

**Comment by macrodesignuk** [comment]
Thank you for your reply, I have to apologise, now that you mention the version, I realised I was testing with an older version 4.0.16, but now that I've switched to the latest 4.4.1, its all working fine.

I did run the chrome browser debugger on the tinymce 4.0.16 whilst trying to establish what was going wrong and it seems that during the link generation, tinymce repaints the screen and deselects the chosen word, which caused it to fail the link insert.

This is a really good plugin and a definite improvement upon the original inspiration grid manager. 

Thanks again
